src/main/java/core/basesyntax/db/FruitTransactionsStorage.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public class FruitTransactionsStorage implements Storage<FruitTransaction> {
```

Comment:
FruitTransaction is not a storage structure. It is a class that help you work with data. Shorage should be a `public static Map<String, Integer>`